nnunet/training/loss_functions/dice_loss.py

```diff
@@ -359,6 +323,7 @@ def __init__(self, soft_dice_kwargs, ce_kwargs, aggregate="sum", square_dice=Fal
         self.weight_ce = weight_ce
         self.aggregate = aggregate
         self.ce = RobustCrossEntropyLoss(**ce_kwargs)
+        self.ce = RobustCrossEntropyLoss(ignore_index=ignore_label if ignore_label else -100, **ce_kwargs)
```

Comment:
You get -100 not only if `ignore_label` is None, but also if it is 0. Is that a problem?